public/frenchPress.json

```diff
@@ -0,0 +1,21 @@
+[{
+    "name": "classic",
+    "title": "Classic", 
+    "notes": "A classic take on French Press coffee.",
+    "ingridients": {
+      "coffee": 56,
+      "water": 235,
+      "grind": 7,
+      "temp": 100,
+      "inverted": true,
```

Comment:
set it to false, it makes icon in a list of recipes upside down

Reply:
_ah_ okay I was stumped about what was causing that 180-degree rotation.